src/common/ceph_argparse.cc

```diff
@@ -113,7 +113,6 @@ void env_to_vec(std::vector<const char*>& args, const char *name)
   std::vector<const char*> env_arguments;
   static vector<string> str_vec;
   std::vector<const char*> env;
-  str_vec.clear();
```

Comment:
`vector::clear()` does **not** release the memory. `vector::shrink_to_fit()` does. and i don't think the memory usage is a concern here.